ubccr/xdmod
# Remove ldif

## Description
Removing functionality that's not being used anymore

## Motivation and Context
Reduce the amount of unused code in the code base.

## Tests performed
Manual Tests:

**Test 1:**
- Login to the Internal Dashboard
- Select 'User Management' -> 'Existing Users' Tab
- Ensure that the 'Generate LDIF' button is no longer visible. 

## Types of changes


## Checklist:
